src/LearningObjectDownload/Interactor.ts

```diff
@@ -215,7 +216,7 @@ async function bundleLearningObject({
     const [license, readMe, files, children] = await Promise.all([
       bundleCCLicense(prefix),
       bundleReadMe({
-        uri: learningObject.materials.pdf.url,
+        uri: learningObject.materials.pdf.url ? learningObject.materials.pdf.url : generatePDFURL(learningObject.id),
```

Comment:
This can be shorted to
```suggestion
        uri: learningObject.materials.pdf.url || generatePDFURL(learningObject.id),
```